url.bs

```diff
@@ -768,7 +770,8 @@ no purpose other than being a location the algorithm can jump to.
  <a>IPv6 serializer</a> on <var>host</var>,
  followed by "<code>]</code>".
 
- <li><p>Otherwise, <var>host</var> is a <a>domain</a> or <a>opaque host</a>, return <var>host</var>.
+ <li><p>Otherwise, <var>host</var> is a <a>domain</a>, an <a>opaque host</a>, or an <a>empty
+ host</a>, return <var>host</var>.
```

Comment:
I think it reads more naturally if you leave out the "an". So "A domain, opaque host, or empty host".